### PR TITLE
Create com.github.rajsolai.textsnatcher.yml

### PR DESCRIPTION
**Pull request by RajSolai**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### Other Permissions used
- [--system-talk-name=org.freedesktop.Accounts](https://valadoc.org/granite/Granite.Settings.prefers_color_scheme.html) Granite Needs this permission for prefers_colors_scheme method.

- "--talk-name=org.gnome.Shell.Screenshot" is used to access screenshot dbus-service

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build com.github.rajsolai.textsnatcher

**Comment by barthalion** [comment]
bot, build com.github.rajsolai.textsnatcher

**Comment by barthalion** [comment]
Does not seem to work for me:

```
** (com.github.rajsolai.textsnatcher:2): CRITICAL **: 09:55:37.403: TesseractTrigger.vala:21: GDBus.Error:org.freedesktop.DBus.Error.AccessDenied: SelectArea is not allowed
```

**Comment by RajSolai** [comment]
Yes I too tested on various desktop envs the issue persists Usage of flatpak portal might resolve it. I will commit with the issue resolved, Thanks a lot @barthalion

**Comment by RajSolai** [comment]
Resolved the ```GDBus.Error:org.freedesktop.DBus.Error.AccessDenied``` issue,
requesting review @hfiguiere @barthalion

**Comment by barthalion** [comment]
bot, build com.github.rajsolai.textsnatcher

**Comment by barthalion** [comment]
```
Yes | 00 tested on various desktop envs the issue persists Usage of latpak portal might resolve it. | will commit with the issue
resolved, Thanks a lot @barthalion
```

I confirm it works!

File: com.github.rajsolai.textsnatcher.yml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  - name: libportal
+    buildsystem: meson
+    builddir: true
+    config-opts:
+      - "-Dbackends=gtk3"
+      - "-Ddocs=false"
+    sources:
+      - type: git
+        url: https://github.com/flatpak/libportal.git
+        branch: master
```
Comment (barthalion) [inline com.github.rajsolai.textsnatcher.yml:27]: One thing: this should use a specific commit, instead of a branch, to ensure reproducible builds.

Comment (RajSolai) [reply com.github.rajsolai.textsnatcher.yml:27]: Sure I will change it thanks @barthalion

**Comment by RajSolai** [comment]
Changed the source of libportal from git master branch to release tar file

**Comment by barthalion** [comment]
bot, build com.github.rajsolai.textsnatcher

**Pull request closed without merging**